src/style.css

```
@@ -1,3 +1,72 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  color: black;
+  background-color: #fff;
+  font-size: 12px;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+/* .header * {
+  display: inline;
+} */
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__list-item {
+  margin-right: 20px;
+}
+
+.nav__list-item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 23px 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  background-color: #00acdc;
```

better to move this color to some var and reuse it everywhere

---

what does mean 'some var'?

---

add selector
```
:root {
  --link-color: #00acdc;
}
```


and use this variable everywhere 
```suggestion
  background-color: var(--link-color);
```